src/Symfony/Component/HttpFoundation/StreamedResponse.php

```diff
@@ -59,17 +59,22 @@ public function setCallback(callable $callback): static
     /**
      * This method only sends the headers once.
      *
+     * @param null|positive-int $statusCode The status code to use. Override the statusCode property if set and not null.
```

Comment:
The status code to use,verride the statusCode property if set and not null (not final dot)